bin/filter_sce_rds.R

```diff
@@ -32,6 +32,19 @@ option_list <- list(
     default = FALSE,
     help = "flag to set the miQC's enforce_left_cutoff option to TRUE"
   ),
+  make_option(
+    opt_str = c("--adt_barcode_file"),
+    type = "character",
+    default = NULL,
+    help = "Optional path to an ADT barcode file, where the third column indicates
+      the ADT type (target, negative control, or positive control)."
+  ),
+  make_option(
+    opt_str = c("--adt_name"),
+    type = "character",
+    default = NULL,
```

Comment:
Adding the default. I'm a bit conflicted on this, but I think it is safe as we do have a `opt$adt_name %in% altExpNames(sce)` in there.
```suggestion
    default = "CITEseq",
```